ui-asset-management/api/config.yaml

```
@@ -128,4 +128,4 @@
         requireDynamoDB: true
         requireSecretsManager: true
       layers:
-        - name: svc-asset-count-layer
\ No newline at end of file
+        - name: svc-asset-state-count-layer
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)